File: 2.py
'''
Write a program to print below pattern but accept the number of rows from the user.
* * * *
  * * *
    * *
      *
'''
rows =int(input("Enter Number of rows: "))
i = rows
while i >= 1:
    j = rows
    while j > i:
        # display space
        print(' ', end=' ')
        j -= 1
    k = 1
    while k <= i:
        print('*', end=' ')
        k += 1
    print()
    i -= 1

File: 2.c
// Write a program to print above pattern but accept the number of rows from the user.

#include <stdio.h>

int main()
{
    int count;
    printf("Enter your number: ");
    scanf("%d",&count);
    for (int i = count; i >= 0; i--) {
        for (int k = 0; k < count-i; k++) {
            printf(" ");
        }
        for (int j = 0; j < i; j++) {
            printf("*");
        }
        printf("\n");
    }
}
